index.html

```diff
@@ -1,13 +1,320 @@
 <!DOCTYPE html>
-<html lang="en">
-    <head>
-        <title>Design Teardown</title>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link href="assets/style.css" rel="stylesheet">
-        <link href="assets/reset.css" rel="stylesheet">
-    </head>
-    <body>
-
-    </body>
+<html lang="zxx">
+
+<head>
+    <title>Design Teardown</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="assets/style.css" rel="stylesheet">
+    <link href="assets/reset.css" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,500,500i,700,700i,900,900i&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li class="item1 logo">
+                <a href="#">
+                    Smashing
+                    Magazine
+                </a>
+            </li>
```

Comment:
It uses ```<div>```  to position elements as at the source page
![k1](https://user-images.githubusercontent.com/46388113/74876789-3a64c280-535c-11ea-8e12-32261ffcc409.PNG)